src/DataWrangling/metadata.jl

```diff
@@ -208,49 +220,47 @@ that `metadata` corresponds to.
 vertical_interfaces(metadata::Metadata{V}) where V =
     error("vertical_interfaces not implemented for $V")
 
-variable_is_three_dimensional(metadata::Metadata{V}) where V =
-    error("variable_is_three_dimensional not implemented for $V")
+is_three_dimensional(metadata) = true
 
-function dataset_latitude_extent end
+function latitude_bounds end
 
 """
-    empty_field(metadata::Metadata;
-                architecture = CPU(),
+    empty_field(metadata::Metadata, arch=CPU();
                 horizontal_halo = (7, 7))
 
-Return an empty field of `metadata` on `architecture` and with `horizontal_halo`s.
+Return an empty field of `metadata` on `arch`itecture and with `halo`.
 """
-function empty_field(metadata::Metadata;
-                     architecture = CPU(),
-                     horizontal_halo = (7, 7))
+function empty_field(metadata::Metadata, arch=CPU();
+                     halo = (7, 7, 3))
 
     Nx, Ny, Nz, _ = size(metadata)
     loc = location(metadata)
     longitude = (0, 360)
-    latitude = dataset_latitude_extent(metadata)
+    latitude = latitude_bounds(metadata)
     TX, TY = (Periodic, Bounded)
 
-    if variable_is_three_dimensional(metadata)
+    if is_three_dimensional(metadata)
         TZ = Bounded
         LZ = Center
         z = vertical_interfaces(metadata)
-        halo = (horizontal_halo..., 3)
         sz = (Nx, Ny, Nz)
     else # the variable is two-dimensional
         TZ = Flat
         LZ = Nothing
         z = nothing
-        halo = horizontal_halo
+        halo = halo[1:2]
         sz = (Nx, Ny)
     end
 
-    grid = LatitudeLongitudeGrid(architecture, Float32; halo, longitude, latitude, z,
+    grid = LatitudeLongitudeGrid(arch, Float32;
+                                 halo, longitude, latitude, z,
                                  size = sz,
                                  topology = (TX, TY, TZ))
+
     return Field{loc...}(grid)
 end
 
 struct Celsius end
 struct Kelvin end
 
-function dataset_temperature_units end
+dataset_temperature_units(metadata) = Celsius()
```

Comment:
In #467 I deliberately didn't wanna have a fallback

Why would the default is Celsius? I mean, it does feel natural to me... But my thinking was that whoever adds a dataset needed to ensure that they added a method and they particularly checked the units and added the method accordingly.

Reply:
Do we have any examples of Kelvin temperature ocean datasets? Defaults just help accelerate the process of implementing new datasets. Faster development = better for all

Reply:
Also its very obvious that the units are wrong so the cost of this default is low I think. Defaults have a higher cost when they are subtle

Reply:
Perhaps we don’t need this method at all. People who add datasets just ensure that the temp is in Celsius?

Reply:
EN4 is Kelvin! Touche!

Reply:
kudos to them also. Kelvin is the True Way